dm/worker/worker.go

```diff
@@ -391,7 +426,10 @@ func (w *Worker) doFetchDDLInfo(ctx context.Context, ch chan<- *pb.DDLInfo) {
 	if !ok {
 		return // should not go here
 	}
+
+	w.RLock()
 	w.subTasks[v.Task].SaveDDLInfo(v)
```

Comment:
can `w.subTasks[v.Task]` become `nil`?

Reply:
🤔 The essential problem is that the worker's concurrency model is really a bit bad, isn't it, but this pr hasn't planned to handle this yet